packages/desktop-client/src/components/sidebar.js

```diff
@@ -284,7 +284,12 @@ function Account({
             <AlignedText
               left={name}
               right={
-                <CellValue debug={true} binding={query} type="financial" />
+                <CellValue
+                  debug={true}
+                  binding={query}
+                  type="financial"
+                  privacyFilter
```

Comment:
Do you think it would make sense to default-apply `privacyFilter` if `type` is `financial`?

Reply:
I think this makes sense but doing so will make the privacy filter apply to mobile. I will need to add `privacyFilter={false}` for mobile cells.

Reply:
Maybe entirely disable it for now on mobile? (i.e. return `false` from the hook that checks if privacy mode is enabled)

Reply:
Updated `PrivacyFilter` to not show on narrow width devices.